frontend/src/lib/components/icon/svg/material/warning.ts

```diff
@@ -0,0 +1 @@
+export default 'xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"/>';
```

Comment:
Confirm that `xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">` is actually needed, based on the instructions it should be redundant